### PR TITLE
fix issue points

### PR DESCRIPTION
**Pull request by puhrez**
address #63

### CODE REVIEW

**Comment by puhrez** [comment]
for package distribution we can't reference things outside of this folder, because it's not in the package.

an example stack trace:
``` Running setup.py (path:/private/var/folders/44/rg86bv9578v197smmykx6jvm0000gn/T/pip-build-p1deq_cl/samsara-sdk/setup.py) egg_info for package samsara-sdk
    Running command python setup.py egg_info
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/44/rg86bv9578v197smmykx6jvm0000gn/T/pip-build-p1deq_cl/samsara-sdk/setup.py", line 28, in <module>
        long_description=get_long_description(),
      File "/private/var/folders/44/rg86bv9578v197smmykx6jvm0000gn/T/pip-build-p1deq_cl/samsara-sdk/setup.py", line 13, in get_long_description
        with open('../../docs/clients/python-client.md') as f:
    FileNotFoundError: [Errno 2] No such file or directory: '../../docs/clients/python-client.md'
```

**Comment by BrunoBonacci** [comment]
Well I guess we will have to duplicate the documentation then.

File: docs/clients/python-client.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ tab_bar: clients
...
 ```bash
-$ pip install samsara-sdk
+$ pip install samsara_sdk
 ```
 
 To bring it into the repl:
 
 ```python
-> from samsara-sdk import SamsaraClient
+> from samsara_sdk import SamsaraClient
```
Comment (BrunoBonacci) [inline docs/clients/python-client.md]: didn't you say that this should be `from samsara_sdk.client import SamsaraClient` ??

Comment (puhrez) [inline docs/clients/python-client.md]: yea, but that was only becuase I had forgotten to add [this line](https://github.com/samsara/samsara/pull/64/commits/32734b01f392346e31ca77daff9c09850a2fde7d#diff-3adec5d4a0408c92608059ea9f371a31R1)

**Comment by BrunoBonacci** [comment]
merged thanks

**Pull request merged**